packages/hooks/use-namespace/index.ts

```diff
@@ -62,15 +62,19 @@ export const useNamespace = (block: string) => {
   const cssVar = (object: Record<string, string>) => {
     const styles: Record<string, string> = {}
     for (const key in object) {
-      styles[`--${namespace.value}-${key}`] = object[key]
+      if (object[key]) {
```

Comment:
I remember that vue automatically determines whether it is empty or not.

Of course it's good to add it.